wiremock/mappings/AdjudicationsApi_GetAdjudicationsPage.json

```diff
@@ -0,0 +1,292 @@
+          "description": "(a) Attempts to commit, (b) incites another inmate to commit, or (c) assists another inmate to commit or to attempt to commit, any of the foregoing offences - incite or assist the disobeying of an order"
+        },
+        {
+          "id": "171",
+          "code": "55:29AC",
+          "description": "(a) Attempts to commit, (b) incites another inmate to commit, or (c) assists another inmate to commit or to attempt to commit, any of the foregoing offences - incite the use of threats/abuse/insults"
+        },
+        {
+          "id": "92",
+          "code": "51:18A",
+          "description": "Absents himself from any place he is required to be or is present at any place where he is not authorised to be - absence without permission"
+        },
+        {
+          "id": "79",
+          "code": "51:1F",
+          "description": "Commits any assault - all assault on other person"
+        },
+        {
+          "id": "200",
+          "code": "55:1E",
+          "description": "Commits any assault - all assault on other person"
+        },
+        {
+          "id": "78",
+          "code": "51:1B",
+          "description": "Commits any assault - assault on inmate"
+        },
+        {
+          "id": "2",
+          "code": "51:1A",
+          "description": "Commits any racially aggravated assault"
+        },
+        {
+          "id": "84",
+          "code": "51:2C",
+          "description": "Detains any person against his will - detention against will of prison officer grade"
+        },
+        {
+          "id": "48",
+          "code": "51:22",
+          "description": "Disobeys any lawful order"
+        },
+        {
+          "id": "49",
+          "code": "51:23",
+          "description": "Disobeys or fails to comply with any rule or regulation applying to him"
+        },
+        {
+          "id": "17",
+          "code": "51:21",
+          "description": "Intentionally fails to work properly or, being required to work, refuses to do so"
+        },
+        {
+          "id": "13",
+          "code": "51:16",
+          "description": "Intentionally or recklessly sets fire to any part of a prison or any other property, whether or not his own"
+        },
+        {
+          "id": "18",
+          "code": "51:24",
+          "description": "Receives any controlled drug, or, without the consent of an officer, any other article, during the course of a visit (not being an interview such as is mentioned in rule 38)"
+        },
+        {
+          "id": "16",
+          "code": "51:20",
+          "description": "Uses threatening, abusive or insulting words or behaviour"
+        }
+      ],
+      "agencies": [
+        {
+          "agencyId": "MDI",
+          "description": "Moorland (HMP & YOI)",
+          "active": true
+        },
+        {
+          "agencyId": "WWI",
+          "description": "Wandsworth (HMP)",
+          "active": true
+        }
+      ]
```

Comment:
The `Agency` objects in this list seem to be missing the `agencyType` field as required in `AdjudicationsPage` which causes a 500 response when testing the `GET /people/{crn}/adjudications` endpoint locally. The [Swagger docs for the Adjudications API](https://manage-adjudications-api-dev.hmpps.service.justice.gov.uk/swagger-ui/index.html#/01.%20Adjudication%20Summary/getAdjudicationsByPrisonerNumber) seems to indicate that `agencyType` isn't returned, but it would be good to confirm this is the case so that we can reuse this Wiremock mapping elsewhere (like [ap-tools](https://github.com/ministryofjustice/hmpps-approved-premises-tools)).

Reply:
Hi @danlivings-dxw thanks for digging into this! That's right. When the endpoint was created with https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/414 (cc @Mjwillis) we didn't include `agencyType`

Seems that you, and other consumers of that endpoint don't use the field so seems fine to no longer include it. I've dropped it from `Agency` with https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/892/commits/913908c807445b9a9240002e50ee0f051dcb477e